catt/api.py

```diff
@@ -160,6 +160,15 @@ def volumedown(self, delta: float) -> None:
 
         self.controller.volumedown(delta)
 
+    def volumemute(self, muted: bool) -> None:
+        """
+        Enable mute on supported devices. Added by @neurodiv-eric
```

Comment:
Could you move the credit here to the CONTRIBUTING file? Docstrings should only be for function documentation. The changelog will also show your name automatically.

Reply:
Done! Anything else look off?